### PR TITLE
theme: add experiment, collaboration, affiliation, & no of pages

### LINKED ISSUE #2767: Add fields in detailed page
In the detailed pages, eg. here: https://labs.inspirehep.net/literature/1512924 more fields need to be added:

- [x] add the experiment

- [x] add the collaboration

- [x] add the number of pages

- [x] add the affiliations in parenthesis next to the author name. 
In fact, this already exists when you click on:
show
 you will see all the affiliations in parentheses:
show2

### PR DESCRIPTION
**Pull request by Dinika**
Signed-off-by: Dinika <dinika.saxena@cern.ch>

## Description
Added the collaboration, experiments, number of pages, and affiliations field to the detailed literature template

## Related Issue
fixes #2767 

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Dinika** [comment]
affiliations and number of pages
Affiliations and number of pages

coolab with multiple authors
Collaborations, number of authors > 0

collaboration with 0 authors
Collaborations when number of authors = 0
Also, the experiment field is displayed for this record

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl [hunk 1]
```diff
@@ -49,18 +49,24 @@
...
+                {{ record_arxiv(record) }}
+              {% endif %}
+            </div>
+          {% endif %}
+          {% if record.get('accelerator_experiments') %}
+            <div> Experiment: {{ record_experiment(record) }} </div>
           {% endif %}
-        {% if record.urls or record.external_system_numbers %}
-          <div id="external_links">{{ record_links(record) }}</div>
+          {% if record.urls or record.external_system_numbers %}
```
Comment (jmartinm) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl]: According to your changes in https://github.com/inspirehep/inspire-next/pull/2743/files  `record.external_system_numbers` should be `record.external_system_identifiers`

**Comment by jmartinm** [comment]
Merged in https://github.com/inspirehep/inspire-next/pull/2882

**Comment by Dinika** [comment]
closes #2767

**Pull request closed without merging**